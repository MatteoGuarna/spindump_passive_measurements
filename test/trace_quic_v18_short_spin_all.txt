Like trace_quic_v18_short_spin, but with full reporting of spin values
