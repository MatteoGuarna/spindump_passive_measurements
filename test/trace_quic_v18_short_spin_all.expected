QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-ba5573c827ffe4e911 (49576:4433) at 1553410393050231 new starting packets 0 0 bytes 0 0
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101193 change starting packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101193 measurement up right 50962 packets 1 0 bytes 1228 0 bandwidth 1228 0
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101456 spinvalue up 0 responder packets 1 1 bytes 1228 424 bandwidth 1228 424
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393101983 spinvalue up 0 responder packets 1 2 bytes 1228 754 bandwidth 1228 754
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393103467 spinvalue up 0 responder packets 1 3 bytes 1228 2034 bandwidth 1228 2034
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393110728 spinvalue up 1 initiator packets 2 4 bytes 1372 3314 bandwidth 1372 3314
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393143298 spinvalue up 0 responder packets 3 4 bytes 1457 3314 bandwidth 1457 3314
QUIC 10.0.6.137 <-> 52.58.13.57 1bb4f0a9-3f39d101ab09b290 (49576:4433) at 1553410393143630 spinvalue up 0 responder packets 3 5 bytes 1457 4594 bandwidth 1457 4594
QUIC 10.0.6.137 <-> 52.58.13.57 a6da7373-3f39d101ab09b290 (49576:4433) at 1553410393150096 spinvalue up 1 initiator packets 3 7 bytes 1457 5195 bandwidth 1457 5195
QUIC 10.0.6.137 <-> 52.58.13.57 a6da7373-3f39d101ab09b290 (49576:4433) at 1553410393150737 spinvalue up 1 initiator packets 4 7 bytes 1521 5195 bandwidth 1521 5195
QUIC 10.0.6.137 <-> 52.58.13.57 a6da7373-3f39d101ab09b290 (49576:4433) at 1553410393163945 spinvalue up 1 responder packets 5 7 bytes 1579 5195 bandwidth 1579 5195
QUIC 10.0.6.137 <-> 52.58.13.57 a6da7373-3f39d101ab09b290 (49576:4433) at 1553410393163945 spinflip up 0-1 responder packets 5 7 bytes 1579 5195 bandwidth 1579 5195
QUIC 10.0.6.137 <-> 52.58.13.57 a6da7373-3f39d101ab09b290 (49576:4433) at 1553410393173946 spinvalue up 1 responder packets 5 8 bytes 1579 5256 bandwidth 1579 5256
